web/controllers/character_controller.ex

```diff
@@ -61,6 +61,8 @@ defmodule EdgeBuilder.CharacterController do
 
     render conn, :show,
       title: character.name,
+      description: description_for_character(character, user),
+      og_image_url: character.portrait_url,
```

Comment:
Rather than pass in yet another new argument to `render`, I'd prefer to just pull the url off the character as portrait_url in the show view. This will be an Ecto changeset - look for another example in the show view for the correct syntax to accomplish this.